Published service-bindings v1.0.0 package is not compiled to ES5 fully

The published [service-bindings package](https://www.npmjs.com/package/@nebhale/service-bindings) v1.0.0 does not seem to be fully compiled to ES5. After importing it into a ES6 nodejs project, attempting to import 'bindings' and 'binding' modules throws an error that they are not available:

```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/<user>/workspace/<project>/node_modules/@nebhale/service-bindings/bindings.js'
``` 

After further inspection, it seems like the only ES5 compiled file in the project is `index.js` and the rest are original Typescript. By entering the dependency and running `npm install` followed by `npm run build`, only then does the import work properly since the proper ES5 files are created.

Would you be able to verify that this is an issue? If so, publish a new artifact for consumption on npmjs.com?

--- a/.gitignore
+++ b/.gitignore
@@ -1,6 +1,4 @@
 .nyc_output/
-*.d.ts
-*.js
-*.js.map
 coverage/
+lib/
 node_modules/

--- a/package.json
+++ b/package.json
@@ -1,5 +1,6 @@
 {
   "devDependencies": {
+    "@tsconfig/recommended": "^1.0.1",
     "@types/chai": "^4.2.21",
     "@types/mocha": "^9.0.0",
     "@types/pg": "^8.6.1",
@@ -12,10 +13,16 @@
   },
   "name": "@nebhale/service-bindings",
   "version": "1.0.0",
-  "main": "index.js",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
   "scripts": {
     "build": "tsc",
-    "test": "nyc --reporter lcovonly mocha -r ts-node/register *.spec.ts"
+    "test": "nyc --reporter lcovonly mocha -r ts-node/register *.spec.ts",
+    "prepare": "npm run build",
+    "prepublishOnly": "npm test",
+    "preversion": "npm test",
+    "version": "npm run build && git add .",
+    "postversion": "git push && git push --tags"
   },
   "keywords": [],
   "author": "Ben Hale",
@@ -27,9 +34,9 @@
   "nyc": {
     "exclude": "**/*.spec.ts"
   },
-  "directories": {
-    "example": "example"
-  },
+  "files": [
+    "lib/*"
+  ],
   "repository": {
     "type": "git",
     "url": "git+https://github.com/nebhale/client-nodejs.git"

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -1,20 +1,21 @@
 {
+  "extends": "@tsconfig/recommended/tsconfig.json",
   "compilerOptions": {
     /* Visit https://aka.ms/tsconfig.json to read more about this file */
 
     /* Basic Options */
     // "incremental": true,                         /* Enable incremental compilation */
-    "target": "ES5",                                /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', 'ES2021', or 'ESNEXT'. */
-    "module": "commonjs",                           /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */
+    // "target": "ES5",                             /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', 'ES2021', or 'ESNEXT'. */
+    // "module": "commonjs",                        /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */
     // "lib": [],                                   /* Specify library files to be included in the compilation. */
     // "allowJs": true,                             /* Allow javascript files to be compiled. */
     // "checkJs": true,                             /* Report errors in .js files. */
     // "jsx": "preserve",                           /* Specify JSX code generation: 'preserve', 'react-native', 'react', 'react-jsx' or 'react-jsxdev'. */
-     "declaration": true,                           /* Generates corresponding '.d.ts' file. */
+    "declaration": true,                            /* Generates corresponding '.d.ts' file. */
     // "declarationMap": true,                      /* Generates a sourcemap for each corresponding '.d.ts' file. */
-     "sourceMap": true,                             /* Generates corresponding '.map' file. */
+    "sourceMap": true,                              /* Generates corresponding '.map' file. */
     // "outFile": "./",                             /* Concatenate and emit output to single file. */
-    // "outDir": "./",                              /* Redirect output structure to the directory. */
+    "outDir": "./lib",                              /* Redirect output structure to the directory. */
     // "rootDir": "./",                             /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
     // "composite": true,                           /* Enable project compilation */
     // "tsBuildInfoFile": "./",                     /* Specify file to store incremental compilation information */
@@ -26,7 +27,7 @@
 
     /* Strict Type-Checking Options */
     "strict": true,                                 /* Enable all strict type-checking options. */
-    // "noImplicitAny": true,                       /* Raise error on expressions and declarations with an implied 'any' type. */
+    "noImplicitAny": true,                          /* Raise error on expressions and declarations with an implied 'any' type. */
     // "strictNullChecks": true,                    /* Enable strict null checks. */
     // "strictFunctionTypes": true,                 /* Enable strict checking of function types. */
     // "strictBindCallApply": true,                 /* Enable strict 'bind', 'call', and 'apply' methods on functions. */
@@ -35,10 +36,10 @@
     // "alwaysStrict": true,                        /* Parse in strict mode and emit "use strict" for each source file. */
 
     /* Additional Checks */
-     "noUnusedLocals": true,                        /* Report errors on unused locals. */
-     "noUnusedParameters": true,                    /* Report errors on unused parameters. */
-     "noImplicitReturns": true,                     /* Report error when not all code paths in function return a value. */
-     "noFallthroughCasesInSwitch": true,            /* Report errors for fallthrough cases in switch statement. */
+    "noUnusedLocals": true,                         /* Report errors on unused locals. */
+    "noUnusedParameters": true,                     /* Report errors on unused parameters. */
+    "noImplicitReturns": true,                      /* Report error when not all code paths in function return a value. */
+    "noFallthroughCasesInSwitch": true,             /* Report errors for fallthrough cases in switch statement. */
     // "noUncheckedIndexedAccess": true,            /* Include 'undefined' in index signature results */
     // "noImplicitOverride": true,                  /* Ensure overriding members in derived classes are marked with an 'override' modifier. */
     // "noPropertyAccessFromIndexSignature": true,  /* Require undeclared properties from index signatures to use element accesses. */
@@ -49,11 +50,11 @@
     // "paths": {},                                 /* A series of entries which re-map imports to lookup locations relative to the 'baseUrl'. */
     // "rootDirs": [],                              /* List of root folders whose combined content represents the structure of the project at runtime. */
     // "typeRoots": [],                             /* List of folders to include type definitions from. */
-     "types": [                                     /* Type declaration files to be included in compilation. */
-       "node"
-     ],
+    "types": [                                      /* Type declaration files to be included in compilation. */
+      "node"
+    ],
     // "allowSyntheticDefaultImports": true,        /* Allow default imports from modules with no default export. This does not affect code emit, just typechecking. */
-    "esModuleInterop": true,                        /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */
+    // "esModuleInterop": true,                        /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */
     // "preserveSymlinks": true,                    /* Do not resolve the real path of symlinks. */
     // "allowUmdGlobalAccess": true,                /* Allow accessing UMD globals from modules. */
 
@@ -68,7 +69,8 @@
     // "emitDecoratorMetadata": true,               /* Enables experimental support for emitting type metadata for decorators. */
 
     /* Advanced Options */
-    "skipLibCheck": true,                           /* Skip type checking of declaration files. */
-    "forceConsistentCasingInFileNames": true        /* Disallow inconsistently-cased references to the same file. */
-  }
+    // "skipLibCheck": true,                        /* Skip type checking of declaration files. */
+    // "forceConsistentCasingInFileNames": true     /* Disallow inconsistently-cased references to the same file. */
+  },
+  "exclude": ["example", "lib", "node_modules", "**/*.spec.ts"]
 }

